src/main/resources/application-test.properties

```diff
@@ -4,6 +4,8 @@ spring.cloud.consul.enabled=false
 spring.sleuth.enabled=false
 #rabbitmq
 spring.rabbitmq.template.exchange=universe-simulator
+#redis cache
+cache.ttl.seconds=60
```

Comment:
After you move the ttl property to `application.properties`, there won't be any more need to add it to test properties. It will be picked up automatically when running tests, because all profiles inherit properties from `application.properties`.

Reply:
Done